### PR TITLE
Hamburg AngularJS Meetup

### PR DESCRIPTION
**Pull request by web-dave**
Please add us to this list.

### CODE REVIEW

File: data/supporters.json [hunk 1]
```diff
@@ -1008,7 +1008,7 @@
       "twitter": "angular_hamburg",
       "contacts": [
         {
-          "name": "David Müllerchen",
+          "name": "David Muellerchen",
```
Comment (web-dave) [inline data/supporters.json]: no  issues here. just automatism
;)

Comment (PragTob) [inline data/supporters.json]: do you want to chane it or shall we merge?

**Comment by web-dave** [comment]
Please merge. THX

**Comment by hagenburger** [comment]
Done!

**Pull request merged**